docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+services:
+  dynamodb-local:
+    image: amazon/dynamodb-local:latest
+    ports:
+      - "18079:8000"
+    command: ["-jar", "DynamoDBLocal.jar", "-sharedDb", "-inMemory"]
+  dynamoInit:
+    depends_on:
+      - dynamodb-local
+    image: banst/awscli
+    environment:
+      AWS_ACCESS_KEY_ID: 'DUMMYIDEXAMPLE'
+      AWS_SECRET_ACCESS_KEY: 'DUMMYEXAMPLEKEY'
+    command:
+      dynamodb describe-limits --endpoint-url http://dynamodb-local:8000 --region us-west-2
+  mysql-database:
+    image: mysql:8
+    restart: always
+    ports:
+      - "13306:3306"
+    environment:
+      MYSQL_ROOT_PASSWORD: balrog
+      MYSQL_USER: gandalf
+      MYSQL_PASSWORD: flyyoufools
+      MYSQL_DATABASE: cribbage
+    command: ['mysqld', '--skip-character-set-client-handshake', '--character-set-server=utf8mb4', '--collation-server=utf8mb4_0900_as_cs']
+    healthcheck:
+      test: "mysql -P13306 -ugandalf -pflyyoufools  -Dcribbage -e 'SHOW TABLES;'"
+      timeout: 20s
+      retries: 10
+  cribbage-server:
+    image: joshprzybyszewski/cribbage:latest
```

Comment:
🔮🤔 Not for this PR, but I wonder if we'd benefit from splitting our docker image into server/client so it's easier/more convenient/quicker when you want to rebuild just the server or just the client during development

Reply:
in theory, docker caching provides this for us. If we're breaking cache when we shouldn't, then we've incurred unnecessary costs and should re-evaluate the docker file

Reply:
See #99 for why we might want to do this for other reasons too :/